src/main/java/lotto/model/Lotto.java

```diff
@@ -0,0 +1,62 @@
+package lotto.model;
+
+import camp.nextstep.edu.missionutils.Randoms;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Lotto {
+    private static final int START = 1;
+    private static final int END = 45;
+    private static final int SIZE = 6;
+
+    private static final String INVALID_RANGE = "[ERROR] 로또 번호는 1-45 사이의 정수여야 합니다.";
+    private static final String INVALID_NUMBER = "[ERROR] 로또 번호의 모든 숫자는 서로 다른 숫자여야 합니다.";
+    private static final String INVALID_SIZE = "[ERROR] 로또 번호는 6개여야 합니다.";
+
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        validateRange(numbers);
+        validateDuplication(numbers);
+        validateSize(numbers);
+        this.numbers = numbers;
+    }
+
+    public static void validateRange(List<Integer> numbers) {
+        for (Integer number : numbers) {
+            if (number < START || number > END) {
+                throw new IllegalArgumentException(INVALID_RANGE);
+            }
+        }
+    }
+
+    public static void validateDuplication(List<Integer> numbers) {
+        for (Integer number : numbers) {
+            if (Collections.frequency(numbers, number) != 1) {
+                throw new IllegalArgumentException(INVALID_NUMBER);
+            }
+        }
+    }
+
+    public static void validateSize(List<Integer> numbers) {
+        if (numbers.size() != SIZE) {
+            throw new IllegalArgumentException(INVALID_SIZE);
+        }
+    }
+
+    public static List<List<Integer>> makeRandomNumbers(int count) {
+        List<List<Integer>> lottos = new ArrayList<>();
+        for (int i = 0; i < count; i++) {
+            List<Integer> randomNumbers = Randoms.pickUniqueNumbersInRange(START, END, SIZE);
+            Collections.sort(randomNumbers);
+            lottos.add(randomNumbers);
+        }
+        return lottos;
+    }
+
+    public List<Integer> getNumbers() {
+        return numbers;
+    }
```

Comment:
`Lotto`와 같은 일급 컬랙션에서는 getter를 구현할때 외부에서 값을 조작하는것을 막기 위해서 새로운 컬랙션을 만들어서 반환하는것이 좋아요!
```suggestion
    public List<Integer> getNumbers() {
        return new ArrayList<>(this.numbers);
    }
```
위와 같은 방법으로 바꾸는게 좋을 것 같습니다!!
혹시 일급 컬랙션에 대해서 잘 모르신다면 한번 공부해보는것도 추천드릴게요!!

Reply:
일급 컬랙션에 대해 잘 모르고 있었는데 한번 공부해보겠습니다 😊